src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListContent.tsx

```diff
@@ -34,14 +36,47 @@ const sortOptions: SortOptions = [
 ]
 const defaultSort = sortOptions[0].value
 
+function isContentOutOfView() {
+  const element = document.querySelector("#JUMP--artworksGrid")
+  if (element === null) return false
+
+  let { top } = element.getBoundingClientRect()
+  let viewportH = window.innerHeight || document.documentElement.clientHeight
+  return top >= viewportH
+}
+
 const ArtworkListContent: FC<ArtworkListContentProps> = ({ me, relay }) => {
   const { match } = useRouter()
 
+  const { jumpTo } = useJump()
+  const {
+    artworkListItemHasBeenTouched,
+    artworkListHasBeenScrolled,
+    setArtworkListHasBeenScrolled,
+  } = useArtworkListVisibilityContext()
+
   const artworkList = me.artworkList!
   const counts: Counts = {
     artworks: artworkList.artworks?.totalCount ?? 0,
   }
 
+  useEffect(() => {
+    const shouldScroll =
+      isContentOutOfView() &&
+      artworkListItemHasBeenTouched &&
+      !artworkListHasBeenScrolled
```

Comment:
I think we should reset these values (`artworkListItemHasBeenTouched`, `artworkListHasBeenScrolled`), otherwise scrolling won't work when clicking on another artwork list and scrolling happened before


https://github.com/artsy/force/assets/3513494/6017f68f-3293-48ab-8636-d9896c8612af

Reply:
Yeah, I added `artworkListHasBeenScrolled` deliberately to prevent auto-scrolling more than once — but if we don't want to prevent it, I can just revert 50ad5746630ac5b0c134368888667e23c0c767db and we won't see the behavior above.

(`artworkListItemHasBeenTouched` was also intentionally set once based on [this feedback](https://artsy.slack.com/archives/C9SATFLUU/p1686155549672359?thread_ts=1686154582.647879&cid=C9SATFLUU) from @dblandin. I think this is working as intended?)

I'll leave this for now, in case there's more feedback.

Reply:
> I think it would be a good idea to prevent that auto-scroll behavior until a user has interacted with the rail

I think Devon meant not to do scrolling on the initial render (when the user just navigated to the screen, but did not click on any artwork list card in the rail.) But every time when user clicks on the artwork list card in the rail, we should do scrolling

@dblandin is it correct? Did I understand you correctly?

Reply:
I guess I had in mind the kind of accessibility advice that cautions against  [excessive movement that is not controlled by the user](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html), but in fact: this interaction does not violate that, and arguably is controlled by the user. So maybe I'm being too cautious here.

I'll revert the `!artworkListHasBeenScrolled` requirement in https://github.com/artsy/force/commit/50ad5746630ac5b0c134368888667e23c0c767db, and then merge 👍🏽

Reply:
Doh! I forgot to actually commit that revert, added via follow-up PR in #12517